build.sbt

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-import uk.gov.hmrc.DefaultBuildSettings._
-import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
 import play.sbt.routes.RoutesKeys
 import sbt.Tests.{Group, SubProcess}
+import uk.gov.hmrc.DefaultBuildSettings._
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
```

Comment:
You can get rid of this bottom import. It was only used by the line you removed below

Reply:
Deleted line